src/app/shared/abstracts/base-product/base-product.component.ts

```diff
@@ -44,6 +45,12 @@ export abstract class BaseProductComponent {
         return this._weightOptions.map(w => w as string);
     }
 
+    get flavours(): string[] {
+        return this.product.productType === "coffee"
+            ? (this.product as Coffee).tastingNotes ?? []
+            : [];
+    }
+
```

Comment:
This solution will probably need to be adjusted when merging with the base product refactor.